odoo/addons/base/models/ir_module.py

```diff
@@ -313,10 +313,8 @@ def _check_external_dependencies(terp):
         if not depends:
             return
         for pydep in depends.get('python', []):
-            try:
-                importlib.import_module(pydep)
```

Comment:
Can you please keep it, to be retro-compatible?

Reply:
@KangOl changed your mind? https://github.com/odoo/odoo/issues/25541#issuecomment-401327631 (which is not a problem, just want to make sure)

I'm notre how ot make it retro compatible while using the same key. Any suggestion?

Reply:
maybe something like this?

```python
try:
    import pkg_resources.get_distribution as import_module
except:
    import importlib.import_module
```
and
```python
try:
    import_module(pydep)
except ImportError:
    raise ImportError('No module named %s' % (pydep,))
```
and maybe adding a second <code>except</code> for the <code>pkg_resources.DistributionNotFound</code> error if it's not the same as the <code>ImportError</code>.

Reply:
@KangOl  @mreficent 
The PR is for master a retro-compatibility is not required like other changes that master has without retro-compatibility.
If you like a backporting to 11.0 or another version then you will need a retro-compatibility.
Use the same key for supporting both ways is not a good idea since that you can use this metadata for install it